### PR TITLE
modify orchestration related functions in sharding-proxy module and sharding-orchestration module

### PR DESCRIPTION
**Pull request by wgy8283335**
Fixes #3185.

Changes proposed in this pull request:
- modify sharding-orchestration-center
- modify sharding-proxy-backend
- modify sharding-proxy-bootstrap
- modify sharding-proxy-common

### CODE REVIEW

**Comment by wgy8283335** [comment]
\run

File: sharding-orchestration/sharding-orchestration-center/pom.xml [hunk 1]
```diff
@@ -54,6 +54,12 @@
         <dependency>
             <groupId>org.apache.curator</groupId>
             <artifactId>curator-test</artifactId>
-        </dependency>        
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-math</artifactId>
+            <version>2.2</version>
```
Comment (sunbufu) [inline sharding-orchestration/sharding-orchestration-center/pom.xml]: In my view, Set version in parent pom is better way.

File: sharding-orchestration/sharding-orchestration-center/src/main/java/org/apache/shardingsphere/orchestration/center/yaml/swapper/OrchestrationConfigurationYamlSwapper.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+     * Swap from InstanceConfiguration to YamlInstanceConfiguration.
+     *
+     * @param data data to be swapped
+     * @return YAML instance configuration
+     */
+    @Override
+    public YamlOrchestrationConfiguration swap(final OrchestrationConfiguration data) {
+        Map<String, YamlInstanceConfiguration> yamlInstanceConfigurationMap = new HashMap();
+        Map<String, InstanceConfiguration> instanceConfigurationMap = data.getInstanceConfigurationMap();
+        for (Entry<String, InstanceConfiguration> entry : instanceConfigurationMap.entrySet()) {
```
Comment (sunbufu) [inline sharding-orchestration/sharding-orchestration-center/src/main/java/org/apache/shardingsphere/orchestration/center/yaml/swapper/OrchestrationConfigurationYamlSwapper.java]: Variables in the loop structure are named with `each`.

File: sharding-orchestration/sharding-orchestration-center/src/main/java/org/apache/shardingsphere/orchestration/center/yaml/swapper/OrchestrationConfigurationYamlSwapper.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+     * Swap from YamlInstanceConfiguration to InstanceConfiguration.
+     *
+     * @param yamlConfiguration YAML instance configuration
+     * @return swapped object
+     */
+    @Override
+    public OrchestrationConfiguration swap(final YamlOrchestrationConfiguration yamlConfiguration) {
+        Map<String, InstanceConfiguration> instanceConfigurationMap = new HashMap();
+        Map<String, YamlInstanceConfiguration> yamlInstanceConfigurationMap = yamlConfiguration.getInstanceConfigurationMap();
+        for (Entry<String, YamlInstanceConfiguration> entry : yamlInstanceConfigurationMap.entrySet()) {
```
Comment (sunbufu) [inline sharding-orchestration/sharding-orchestration-center/src/main/java/org/apache/shardingsphere/orchestration/center/yaml/swapper/OrchestrationConfigurationYamlSwapper.java]: Variables in the loop structure are named with `each`.

File: sharding-proxy/sharding-proxy-common/pom.xml [hunk 4]
```diff
@@ -34,14 +34,19 @@
...
-        
         <dependency>
             <groupId>com.zaxxer</groupId>
             <artifactId>HikariCP-java7</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.shardingsphere</groupId>
+            <artifactId>sharding-orchestration-center</artifactId>
+            <version>4.0.0-RC3-SNAPSHOT</version>
```
Comment (sunbufu) [inline sharding-proxy/sharding-proxy-common/pom.xml]: This version can use `${project.version}`.

**Pull request merged**